WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -51,7 +51,7 @@ struct HubMenu: View {
                         HubMenuElement(image: menu.icon,
                                        imageColor: menu.iconColor,
                                        text: menu.title,
-                                       badge: 0,
+                                       badge: menu.badge,
```

Comment:
Good point! They mean that the badge is always zero because the notifications count is not working. To keep that behavior, we add this property to the view model in this PR:
```
var badge: HubMenuBadgeType {
            .number(number: 0)
        }
```

If the badge were not zero, it would look the same as before:

<img src="https://user-images.githubusercontent.com/1864060/182584834-35c02273-286f-4416-a43c-a08277da4d57.png" width="375">

Reply:
Thanks for the clarification!